### ISSUE TITLE
[Settings]: Incorrect keyboard focus is observed when naviagting under 'Color' pop-up.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/terminal#15982]
### Windows Terminal version

1.19.2481.0

### Windows build number

23529.1000

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 23529.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3. Set shortcut for 'Set tab color'.
4. Now navigate back to the Window terminal page and Open color pop-up using shortcut.
5. Navigate over the controls and observe the issue.

**User Experience: **
Users who rely on keyboard expect a sequential focus order that is top to down and left to right, if that is not followed then users might get confused and can miss the control.

MAS Reference:
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

**Attachment:**
[Incorrect keyboard focus is observed when naviagting under 'Color' pop-up..zip](https://github.com/microsoft/terminal/files/12644604/Incorrect.keyboard.focus.is.observed.when.naviagting.under.Color.pop-up.zip)

### Expected Behavior

Correct keyboard focus should be there when navigating under 'Color' pop-up. Focus order should follow left to right and top to down approach.

### Actual Behavior

Incorrect keyboard focus is observed when navigating under 'Color' pop-up.

**Comment by carlos-zamora** [comment]
Thanks for filing! This is actually a bug on the ColorPicker itself! I've repro'd it in the WinUI 2 Gallery.
Filed the bug upstream: https://github.com/microsoft/microsoft-ui-xaml/issues/8883

**Comment by carlos-zamora** [comment]
I'll close the bug since it's an external issue, but let me know if the process should be different. Thanks!
